packages/elements/src/components/TextArea/index.tsx

```diff
@@ -10,6 +10,7 @@ export interface TextAreaProps {
   dataTest?: string
   validate?: (value: string) => string | null
   required?: boolean
+  containerClassName?: string
```

Comment:
Can you use className for consistency please https://github.com/reapit/foundations/blob/master/packages/elements/src/components/Layout/index.tsx#L4